templates/terraform/resource.erb

```diff
@@ -199,7 +199,17 @@ func resource<%= resource_name -%>Create(d *schema.ResourceData, meta interface{
     var project string
     if parts := regexp.MustCompile(`projects\/([^\/]+)\/`).FindStringSubmatch(url); parts != nil {
         project = parts[1]
-    }
+    } else {
+
+        // Send the billing_project ID in the X-Goog-User-Project header.
+        config.UserProjectOverride = true
```

Comment:
This was used in cloud feed due to a unique quirk of the cloud feed resource, and we can omit it here. Cloud Feed _requires_ a billing project to function at all, as I understand, so we make sure to send one all the time there.

`sendRequest` will decide to ignore the quota project argument if `config.UserProjectOverride` is false based on https://www.terraform.io/docs/providers/google/guides/provider_reference.html#user_project_override-1. We're expected to send `project` on every `sendRequest` call if there's one available.